configs/talon.json

```
@@ -61,6 +61,7 @@
     ]
   },
   "js_render": true,
+  "js_wait": 3,
```

```suggestion
  "js_wait": 1,
```

3 seems really long for content to load on an headless mode, we usually only need 1s per pages